db/db.go

```diff
@@ -115,6 +115,7 @@ func (c *Client) send(
 	if c.authOverride != "" {
 		opts = append(opts, internal.WithQueryParam(authVarOverride, c.authOverride))
 	}
+	opts = append(opts, internal.WithHeader("X-Firebase-Decoding", "1"))
```

Comment:
Can you also update the tests by adding something to https://github.com/firebase/firebase-admin-go/blob/master/db/db_test.go#L272?

And ideally add a test case to `integration/db/db_test.go` as well?